### PR TITLE
added repeatable header for overflow tables

### PR DESCRIPTION
**Pull request by Chemmic**
Hey,

I've added a new class called "OverflowOnSamePageRepeatableHeaderTableDrawer". The name is kinda long, maybe it should be changed.

I kinda mixed @vandeseer's "RepeatedHeaderTableDrawer" and "OverflowOnSamePageTableDrawer" together.
It allows tables which overflows on the same page to have repeatable headers as well.

PS: the table starting at a different y position was my mistake and no bug. I misunderstood how the offset worked :)

### CODE REVIEW

File: src/test/java/org/vandeseer/integrationtest/OverflowOnSamePageRepeatableHeaderTableDrawerTest.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import org.vandeseer.TestUtils;
+import org.vandeseer.easytable.OverflowOnSamePageRepeatableHeaderTableDrawer;
+import org.vandeseer.easytable.structure.Row;
+import org.vandeseer.easytable.structure.Table;
+import org.vandeseer.easytable.structure.cell.TextCell;
+
+import de.redsix.pdfcompare.CompareResult;
+import de.redsix.pdfcompare.PdfComparator;
+
+public class OverflowOnSamePageRepeatableHeaderTableDrawerTest {
```
Comment (vandeseer) [inline src/test/java/org/vandeseer/integrationtest/OverflowOnSamePageRepeatableHeaderTableDrawerTest.java:23]: This class doesn't test anything yet.
Interestingly I also can't open the new PDF file.

Essentially it's missing the comparing part, like this:
```
final CompareResult compareResult = new PdfComparator<>(
                getExpectedPdfFor(OVERFLOW_ON_SAME_PAGE_PDF),
                getActualPdfFor(OVERFLOW_ON_SAME_PAGE_PDF)
        ).compare();

        assertTrue(compareResult.isEqual());
```

**Review by vandeseer — CHANGES_REQUESTED** [review]
Thanks for the PR! However, I think it needs a little :heart: still. As soon as the test passes (and the PDF itself is readable correctly), I'm happy to merge. (We can also check then if we can easily refactor the code to reduce the duplication, but I think it's best to start like this and improve only afterwards.)

File: src/test/java/org/vandeseer/integrationtest/OverflowOnSamePageRepeatableHeaderTableDrawerTest.java (hunk 1 above)
Comment (Chemmic) [reply src/test/java/org/vandeseer/integrationtest/OverflowOnSamePageRepeatableHeaderTableDrawerTest.java:23]: I'm not sure if you mean something else, but I'm comparing the pdfs at line 45 :)

**Comment by Chemmic** [comment]
Hey,
Thank you for the review!
I don't know what happened exactly, but I couldn't open the PDF as well, it was corrupted somehow.
However, I uploaded a new PDF (working for me now at least)

- Matteo

File: src/test/java/org/vandeseer/integrationtest/OverflowOnSamePageRepeatableHeaderTableDrawerTest.java (hunk 1 above)
Comment (vandeseer) [reply src/test/java/org/vandeseer/integrationtest/OverflowOnSamePageRepeatableHeaderTableDrawerTest.java:23]: Indeed! My bad, I simply overlooked it. Sorry!

**Review by vandeseer — APPROVED** [review]
I will release after the merge (I hope the release process still works as in the past!)

**Comment by Chemmic** [comment]
> > Hey, Thank you for the review! I don't know what happened exactly, but I couldn't open the PDF as well, it was corrupted somehow. However, I uploaded a new PDF (working for me now at least)
> > 
> > * Matteo
> 
> Hey Matteo,
> 
> the new PDF didn't work for me either, but that's not a big thing: I could produce it by running the test and that one worked correctly. I will simply replace it after I merged this PR.
> 
> Thanks again for your contribution! Let me know if you're interested in helping me maintaining this library – I'm open for it, since I don't want the project to die, but I really lack time to properly maintain it myself ...
> 
> Best, Stefan

Hey Stefan,

Sure, I'd be glad to help if I can!
I'm currently working on my computer science bachelor's thesis, so I'll also be somewhat limited in time over the next few months as well.
Feel free to contact me at Matteo.staar@gmx.de if there's anything that needs clarification or if I need to know something, like how the build process works or something like that :)

Best, Matteo

**Pull request merged**